src/app/create/codebases/services/github.service.ts

```diff
@@ -22,26 +20,26 @@ import {
  */
 @Injectable()
 export class GitHubService {
-
-  private static readonly HEADERS = new Headers({
-    'Content-Type': 'application/json',
-    'Accept': 'application/vnd.github.v3+json'
-  });
-
   private cache: Map<string, Observable<any>>;
   private context: Context;
   private gitHubUrl: string;
-
+  private headers = new Headers({
+    'Content-Type': 'application/json',
+    'Accept': 'application/vnd.github.v3+json'
+  });
 
   constructor(
-    private contexts: Contexts,
-    private http: Http,
-    private logger: Logger,
-    private authService: AuthenticationService,
-    private userService: UserService) {
+      private contexts: Contexts,
+      private http: Http,
+      private logger: Logger,
+      private authService: AuthenticationService) {
     this.contexts.current.subscribe(val => this.context = val);
     this.gitHubUrl = 'https://api.github.com';
     this.cache = new Map();
+
+    authService.gitHubToken.map(token => {
+      this.headers.set('Authorization', `token ${token}`);
+    });
```

Comment:
I don't think this will work as you don't subscribe to the gitHubToken - if it's getting called, it's by chance...